pages/configuration/configuration-settings.mdx

```diff
@@ -91,6 +91,28 @@ workers simultaneously.
 
 </Callout>
 
+## Experimental Features
```

Comment:
I wanted to read what's new so noticed title case here (we don't use it in docs).